pkg/kritis/metadata/containeranalysis/cache.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2018 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package containeranalysis
+
+import (
+	kritisv1beta1 "github.com/grafeas/kritis/pkg/kritis/apis/kritis/v1beta1"
+	"github.com/grafeas/kritis/pkg/kritis/metadata"
+	"github.com/grafeas/kritis/pkg/kritis/secrets"
+	containeranalysispb "google.golang.org/genproto/googleapis/devtools/containeranalysis/v1alpha1"
+)
+
+// Cache struct defines Cache for container analysis client.
+type Cache struct {
+	client metadata.Fetcher
+	vCache map[string][]metadata.Vulnerability
+	aCache map[string][]metadata.PGPAttestation
+	nCache map[*kritisv1beta1.AttestationAuthority]*containeranalysispb.Note
+}
+
+// NewCache Create a new Cache for container analysis client.
+func NewCache() (*Cache, error) {
+	c, err := New()
+	if err != nil {
+		return nil, err
+	}
+	return &Cache{
+		client: c,
+		vCache: map[string][]metadata.Vulnerability{},
+		aCache: map[string][]metadata.PGPAttestation{},
+		nCache: map[*kritisv1beta1.AttestationAuthority]*containeranalysispb.Note{},
+	}, nil
+}
+
+// GetVulnerabilities gets Package Vulnerabilities Occurrences for a specified image.
+func (c Cache) GetVulnerabilities(image string) ([]metadata.Vulnerability, error) {
```

Comment:
optional: The "Get" in function names is implicit, and makes the name longer, so Go it'd probably be more idiomatic to omit it. That said, it's more important to be consistent with the other parts of the code base, so feel free to ignore this suggestion.

Reply:
done